packages/components/tests/dummy/app/controllers/components/form/textarea.js

```diff
@@ -8,12 +8,21 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 
 export default class FormTextareaController extends Controller {
-  @tracked fieldIsInvalid;
+  @tracked defaultText =
+    'Ut enim ad minim veniam, quis nostrud exercitation ullamco';
+  @tracked customText =
+    'Ut enim ad minim veniam, quis nostrud exercitation ullamco';
+  @tracked withErrorMessage =
+    'Ut enim ad minim veniam, quis nostrud exercitation ullamco';
 
   @action
   noop() {}
 
-  @action onFieldInput(args) {
-    this.fieldIsInvalid = args.currentLength > args.maxLength;
+  get fieldIsInvalid() {
+    return this.withErrorMessage.length > 50;
```

Comment:
probably, to avoid duplication, the `50` value should be stored in a variable and accessed in the template code via `this.xyz`

Reply:
done in 2eb5879